bridge/client/app/app-header/app-header.component.ts

```diff
@@ -33,10 +44,80 @@ export class AppHeaderComponent implements OnInit, OnDestroy {
         );
       }
     });
+
+    this.versionInfoSub = this.dataService.versionInfo.subscribe(versionInfo => {
+      this.versionInfo = versionInfo;
+      if(versionInfo.versionCheckEnabled === null) {
+        this.showVersionCheckInfoDialog();
+      } else if(versionInfo.versionCheckEnabled) {
+        if(semver.valid(versionInfo.keptnVersion)) {
+          if(versionInfo.availableVersions.cli)
+            this.doVersionCheck(versionInfo.keptnVersion, versionInfo.availableVersions.cli.stable, versionInfo.availableVersions.cli.prerelease, "Keptn");
+        } else {
+          versionInfo.keptnVersionInvalid = true;
+        }
+        if(semver.valid(versionInfo.bridgeVersion)) {
+          if(versionInfo.availableVersions.bridge)
+            this.doVersionCheck(versionInfo.bridgeVersion, versionInfo.availableVersions.bridge.stable, versionInfo.availableVersions.bridge.prerelease, "Keptn Bridge");
+        } else {
+          versionInfo.bridgeVersionInvalid = true;
+        }
+      }
+    });
+  }
+
+  doVersionCheck(currentVersion, stableVersions, prereleaseVersions, type) {
+    stableVersions.forEach(stableVersion => {
+      if(semver.lt(currentVersion, stableVersion)) {
+        let genMessage;
+        switch(semver.diff(currentVersion, stableVersion)) {
+          case 'patch':
+            genMessage = (version, type, major, minor) => `New ${type} ${version} available. This is a patch with bug fixes for your current version. For details how to upgrade visit https://keptn.sh/docs/${major}.${minor}.0/installation/upgrade-keptn`;
```

Comment:
Can you please change the link to: `https://keptn.sh/docs/${major}.${minor}.0/operate/upgrade/`